libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -438,16 +438,17 @@ userdata AP_ESC_Telem_Backend::TelemetryData field motor_temp_cdeg int16_t'skip_
 
 singleton AP_ESC_Telem depends HAL_WITH_ESC_TELEM == 1
 singleton AP_ESC_Telem rename esc_telem
-singleton AP_ESC_Telem method get_rpm boolean uint8_t 0 NUM_SERVO_CHANNELS float'Null
-singleton AP_ESC_Telem method get_temperature boolean uint8_t 0 NUM_SERVO_CHANNELS int16_t'Null
-singleton AP_ESC_Telem method get_motor_temperature boolean uint8_t 0 NUM_SERVO_CHANNELS int16_t'Null
-singleton AP_ESC_Telem method get_current boolean uint8_t 0 NUM_SERVO_CHANNELS float'Null
-singleton AP_ESC_Telem method get_voltage boolean uint8_t 0 NUM_SERVO_CHANNELS float'Null
-singleton AP_ESC_Telem method get_consumption_mah boolean uint8_t 0 NUM_SERVO_CHANNELS float'Null
-singleton AP_ESC_Telem method get_usage_seconds boolean uint8_t 0 NUM_SERVO_CHANNELS uint32_t'Null
-singleton AP_ESC_Telem method update_rpm void uint8_t 0 NUM_SERVO_CHANNELS uint16_t'skip_check float'skip_check
-singleton AP_ESC_Telem method update_telem_data void uint8_t 0 NUM_SERVO_CHANNELS AP_ESC_Telem_Backend::TelemetryData uint16_t'skip_check
-singleton AP_ESC_Telem method set_rpm_scale void uint8_t 0 NUM_SERVO_CHANNELS float'skip_check
+singleton AP_ESC_Telem method get_rpm boolean uint8_t 0 ESC_TELEM_MAX_ESCS float'Null
+singleton AP_ESC_Telem method get_temperature boolean uint8_t 0 ESC_TELEM_MAX_ESCS int16_t'Null
+singleton AP_ESC_Telem method get_motor_temperature boolean uint8_t 0 ESC_TELEM_MAX_ESCS int16_t'Null
+singleton AP_ESC_Telem method get_current boolean uint8_t 0 ESC_TELEM_MAX_ESCS float'Null
+singleton AP_ESC_Telem method get_voltage boolean uint8_t 0 ESC_TELEM_MAX_ESCS float'Null
+singleton AP_ESC_Telem method get_consumption_mah boolean uint8_t 0 ESC_TELEM_MAX_ESCS float'Null
+singleton AP_ESC_Telem method get_usage_seconds boolean uint8_t 0 ESC_TELEM_MAX_ESCS uint32_t'Null
```

Comment:
I think we can just `'skip_check` for all these that return bool. The result will be null and `AP_ESC_Telem` should deal with it correctly. The problem with using a define is that there is no way for the script to get the value.

Reply:
Do you want all ESC_TELEM_MAX_ESCS to be replaced with 'skip_check. I am not sure of the define implementation and have been following the previous implementation. Not sure if it wil brake anything else. @IamPete1 if you are good with it I will update the PR.

Reply:
@robertlong13 FYI

Reply:
Yeah, we should be good to remove them all.